apps/admin-gui/src/app/shared/components/consent-hubs-list/consent-hubs-list.component.html

```diff
@@ -0,0 +1,67 @@
+<div [hidden]="consentHubs.length === 0 || dataSource.filteredData.length === 0" class="card mt-2">
+  <perun-web-apps-table-wrapper
+    (exportData)="exportData($event)"
+    [tableId]="tableId"
+    [dataLength]="dataSource.filteredData.length"
+    [pageSizeOptions]="pageSizeOptions">
+    <table
+      [dataSource]="dataSource"
+      class="w-100"
+      mat-table
+      matSort
+      matSortActive="id"
+      matSortDirection="asc"
+      matSortDisableClear>
+      <ng-container matColumnDef="id">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'SHARED.COMPONENTS.CONSENT_HUBS_LIST.ID' | translate}}
+        </th>
+        <td *matCellDef="let consentHub" class="static-column-size" mat-cell>{{consentHub.id}}</td>
+      </ng-container>
+      <ng-container matColumnDef="name">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'SHARED.COMPONENTS.CONSENT_HUBS_LIST.NAME' | translate}}
+        </th>
+        <td *matCellDef="let consentHub" mat-cell>{{consentHub.name}}</td>
+      </ng-container>
+      <ng-container matColumnDef="enforceConsents">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'SHARED.COMPONENTS.CONSENT_HUBS_LIST.ENFORCE_CONSENTS' | translate}}
+        </th>
+        <td *matCellDef="let consentHub" mat-cell>
+          <mat-slide-toggle
+            (change)="changeEnforceFlag(consentHub, $event)"
+            [checked]="consentHub.enforceConsents"
+            class="mr-2">
+          </mat-slide-toggle>
+        </td>
+      </ng-container>
+      <ng-container matColumnDef="facilities">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'SHARED.COMPONENTS.CONSENT_HUBS_LIST.FACILITIES' | translate}}
+        </th>
+        <td *matCellDef="let consentHub" mat-cell>
+          <span *ngFor="let facility of consentHub.facilities">
+            {{facility.id}}: {{facility.name}}<br />
```

Comment:
I think we have some (not very strong) convention of showing ids with `#` as prefix. So maybe something like `Name #ID` . Not a must more like a suggestion.